Index: rest/build/tmp/compileJava/source-classes-mapping.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>team.healthtech/rest/RestConfiguration.java\r\n team.healthtech.rest.RestConfiguration\r\nteam.healthtech/rest/api/UserController.java\r\n team.healthtech.rest.api.UserController\r\nteam.healthtech/rest/api/CommentController.java\r\n team.healthtech.rest.api.CommentController\r\nteam.healthtech/rest/api/PatientController.java\r\n team.healthtech.rest.api.PatientController\r\nteam.healthtech/rest/api/PatientAppointmentController.java\r\n team.healthtech.rest.api.PatientAppointmentController\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/rest/build/tmp/compileJava/source-classes-mapping.txt b/rest/build/tmp/compileJava/source-classes-mapping.txt
--- a/rest/build/tmp/compileJava/source-classes-mapping.txt	(revision e2bb1d107fd037ca0661f1091744493faf57007b)
+++ b/rest/build/tmp/compileJava/source-classes-mapping.txt	(date 1627999877943)
@@ -4,7 +4,7 @@
  team.healthtech.rest.api.UserController
 team.healthtech/rest/api/CommentController.java
  team.healthtech.rest.api.CommentController
-team.healthtech/rest/api/PatientController.java
- team.healthtech.rest.api.PatientController
 team.healthtech/rest/api/PatientAppointmentController.java
  team.healthtech.rest.api.PatientAppointmentController
+team.healthtech/rest/api/PatientController.java
+ team.healthtech.rest.api.PatientController
